### PR TITLE
Dev invite system

### PR DESCRIPTION
**Pull request by Mopplikus**
This PR adds a button in the profile screen allowing the user to invite other people to the game. The link to the app will still have to be defined in the .strings resource before deployment. The user can then choose on which platform they wish to share the message (any app on their phone with text redirection support should be compatible out of the box).

### CODE REVIEW

File: app/src/test/java/com/github/campus_capture/bootcamp/ScheduleConstantsTest.java
Comment (Jenjamin3000) [inline app/src/test/java/com/github/campus_capture/bootcamp/ScheduleConstantsTest.java:1]: Looks weird to test all the constants but I think test coverage let use no choice...
For later: would it not be great to have a static import for Assertions.asserEquals ? 
Like: `import static org.junit.jupiter.api.Assertions.assertEquals;`

File: app/src/test/java/com/github/campus_capture/bootcamp/PlaceholderFirebaseInterfaceTest.java
Comment (Jenjamin3000) [inline app/src/test/java/com/github/campus_capture/bootcamp/PlaceholderFirebaseInterfaceTest.java:1]: For later: would it not be great to have a static import for Assertions.asserEquals ?
Like: 
```
import static org.junit.jupiter.api.Assertions.assertEquals;
import static org.junit.jupiter.api.Assertions.assertTrue;
import static org.junit.jupiter.api.Assertions.assertFalse;
```

File: app/src/test/java/com/github/campus_capture/bootcamp/ScoreItemTest.java
Comment (Jenjamin3000) [inline app/src/test/java/com/github/campus_capture/bootcamp/ScoreItemTest.java:1]: Still maybe for later the static import.

**Review by Jenjamin3000 — APPROVED** [review]
Looks great to me! All is working on my side. Great job!

**Comment by Mopplikus** [comment]
@Jenjamin3000 I refactored the imports on the tests as requested.

**Review by Jenjamin3000 — APPROVED** [review]
Perfect! All is OK for me we can merge!

**Comment by Mopplikus** [comment]
@Jenjamin3000 I'll just check the Jacoco coverage before, to be sure that everything is well covered (when I manage to get it working)

**Pull request merged**